Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -4673,20 +4668,19 @@ void Document::run_the_update_intersection_observations_steps(HighResolutionTime
                 intersection_rect = compute_intersection(target, observer);
 
                 // 6. Let targetArea be targetRect’s area.
-                auto target_area = target_rect->width() * target_rect->height();
+                auto target_area = target_rect.width() * target_rect.height();
 
                 // 7. Let intersectionArea be intersectionRect’s area.
-                auto intersection_area = intersection_rect->width() * intersection_rect->height();
+                auto intersection_area = intersection_rect.width() * intersection_rect.height();
```

Comment:
Those can both be simplified to `.size()`

Reply:
`size()` returns a `Size<CSSPixels>`, is that what we want here?

Reply:
The `.area()` should take care of that